Makefile

```diff
@@ -5,7 +5,7 @@ BINARY_NAME      = nri-$(INTEGRATION)
 INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
 CONFIG_DIR       = /etc/newrelic-infra/integrations.d
 GO_FILES        := ./src/
-GOFLAGS			 = -mod=mod
+GOFLAGS			 = -mod=readonly
```

Comment:
What does this achieve?

Reply:
with the mod flag, go will automatically update the go.mod and go.sum behind the scenes when an imported package is not provided by any known module. The readonly flag avoids that. 

https://golang.org/ref/mod#build-commands